.github/workflows/MarkStale.yaml

```diff
@@ -0,0 +1,52 @@
+name: Mark Images Stale
+
+on:
+  workflow_dispatch:
+    inputs:
+      SHAsToMarkStale:
+        description: Enter SHAs to mark stale (comma separated)
+        type: string
+
+permissions:
+  contents: read
+  id-token: write
+
+defaults:
+  run:
+    shell: PowerShell
+
+jobs:
+  MarkmagesStale:
+    runs-on: [ Windows-Latest ]
+    if: github.ref == 'refs/heads/main'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: 'Az CLI login'
+        uses: azure/login@v2
+        with:
+            client-id: ${{ secrets.AZURE_CLIENT_ID }}
+            tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+            subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+            enable-AzPSSession: true
+
+      - name: MarkStale
+        env:
+          digests: ${{ inputs.SHAsToMarkStale }}
+        shell: pwsh
+        run: |
+          $erroractionpreference = "STOP"
+          $digests = "$env:digests".Split(",")

```

Comment:
Is there a need to trim too, in case they come in as "SHA1, <space> SHA2"?

Reply:
good idea